spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe User do
+  let!(:user) { create(:user) }
+  let!(:deck) { create(:deck, user_id: user.id) }
+  let!(:card) { create(:card, deck_id: deck.id) }
+
+  context '.notify_pending_cards' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.